.gitignore

```diff
@@ -0,0 +1 @@
+.vagrant
```

Comment:
if you put that into your global .gitignore you never have to add it to other projects